src/engraving/libmscore/measure.cpp

```diff
@@ -3140,85 +3140,16 @@ QString Measure::accessibleInfo() const
+//  LEGACY: this method used to be called stretchMeasure() and was used to
+//  distribute the remaining space at the end of a system. That task is now
+//  performed elsewhere and only the layout tasks are kept.
 //-----------------------------------------------------------------------------
 
-void Measure::stretchMeasure(qreal targetWidth)
+void Measure::layoutMeasureElements()
 {
-    bbox().setWidth(targetWidth);
-
-    Fraction minTick = computeTicks();
-
-    //---------------------------------------------------
-    //    compute stretch
-    //---------------------------------------------------
-
-    std::multimap<qreal, Segment*> springs;
-
-    Segment* seg = first();
-    while (seg && (!seg->enabled() || seg->allElementsInvisible())) {
-        seg = seg->next();
-    }
-    qreal minimumWidth = seg ? seg->x() : 0.0;
-    for (Segment& s : m_segments) {
-        if (!s.enabled() || !s.visible() || s.allElementsInvisible()) {
-            continue;
-        }
-        Fraction t = s.ticks();
-        if (t.isNotZero()) {
-            qreal str = 1.0 + 0.865617 * log(qreal(t.ticks()) / qreal(minTick.ticks()));       // .6 * log(t / minTick.ticks()) / log(2);
-            qreal d   = s.width() / str;
-            s.setStretch(str);
-            springs.insert(std::pair<qreal, Segment*>(d, &s));
-        }
-        minimumWidth += s.width();
-    }
-
-    //---------------------------------------------------
-    //    compute 1/Force for a given Extend
-    //---------------------------------------------------
-
-    if (targetWidth > minimumWidth) {
-        qreal force = 0;
-        qreal c     = 0.0;
-        for (auto i = springs.begin(); i != springs.end();) {
-            c            += i->second->stretch();
-            minimumWidth -= i->second->width();
-            qreal f       = (targetWidth - minimumWidth) / c;
-            ++i;
-            if (i == springs.end() || f <= i->first) {
-                force = f;
-                break;
-            }
-        }
-
-        //---------------------------------------------------
-        //    distribute stretch to segments
-        //---------------------------------------------------
-
-        for (auto& i : springs) {
-            qreal width = force * i.second->stretch();
-            if (width > i.second->width()) {
-                i.second->setWidth(width);
-            }
-        }
-
-        //---------------------------------------------------
-        //    move segments to final position
-        //---------------------------------------------------
-
-        Segment* s = first();
-        while (s && (!s->enabled() || s->allElementsInvisible())) {
-            s = s->next();
-        }
-        qreal x = s ? s->pos().x() : 0.0;
-        while (s) {
-            s->rxpos() = x;
-            x += s->width();
-            s = s->nextEnabled();
-        }
-    }
+    //bbox().setWidth(targetWidth); // Doesn't seem to be needed but I keep it here just to be safe
```

Comment:
as I said previously

always prefer to remove the code if you think it's redundant, if necessary we can recover logic from git history

Reply:
Sorry, forgot that one